app/controllers/api/thresholds_controller.rb

```diff
@@ -21,7 +21,7 @@ class ThresholdsController < BaseController
     respond_to :json
 
     def show
-      respond_with Stream.thresholds(params[:id], params[:unit_symbol])
+      render json: Stream.thresholds(params[:id], params[:unit_symbol])
```

Comment:
This way it always returns json. The alternatives were
- add "accept" json to the client (since this controller is supposed to do json only this seems skippable)
- adding .json in the request url (the route uses a regexp that would consider `.json` as part of the sensor id)